### PR TITLE
CLOUDSTACK-9899 Url validation disabling

### PR DESCRIPTION
**Pull request by DaanHoogland**

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by borisstoyanov** [comment]
@blueorangutan package

**Comment by borisstoyanov** [comment]
@blueorangutan test

**Comment by borisstoyanov** [comment]
@blueorangutan test

**Comment by cloudmonger** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 831
 Hypervisor xenserver
 NetworkType Advanced
 Passed=107
 Failed=6
 Skipped=12

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/r2si930m8xxzavs/AAAzNrnoF1fC3auFrvsKo_8-a?dl=0

**Failed tests:**
* test_volumes.py

 * test_06_download_detached_volume Failing since 2 runs

* test_routers_network_ops.py

 * test_01_isolate_network_FW_PF_default_routes_egress_true Failing since 21 runs

 * test_02_isolate_network_FW_PF_default_routes_egress_false Failing since 21 runs

 * test_01_RVR_Network_FW_PF_SSH_default_routes_egress_true Failing since 21 runs

 * test_02_RVR_Network_FW_PF_SSH_default_routes_egress_false Failing since 21 runs

 * test_03_RVR_Network_check_router_state Failed


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_01_verify_libvirt
test_02_verify_libvirt_after_restart
test_03_verify_libvirt_attach_disk
test_04_verify_guest_lspci
test_05_change_vm_ostype_restart
test_06_verify_guest_lspci_again
test_static_role_account_acls
test_11_ss_nfs_version_on_ssvm
test_nested_virtualization_vmware
test_3d_gpu_support
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_vm_snapshots.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_loadbalance.py
test_routers.py
test_reset_vm_on_reboot.py
test_deploy_vms_with_varied_deploymentplanners.py
test_network.py
test_router_dns.py
test_non_contigiousvlan.py
test_login.py
test_deploy_vm_iso.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_metrics_api.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_vm_life_cycle.py
test_disk_offerings.py

**Comment by rhtyd** [comment]
@DaanHoogland please squash the changes and use the JIRA id in the commit message and not CE ids.

**Comment by wido** [comment]
Could you take a look at @rhtyd his comment @DaanHoogland ? We can then give this one a LGTM

**Comment by rhtyd** [comment]
@wido can you have a look at Travis failures.

File: server/src/com/cloud/storage/VolumeApiServiceImpl.java [hunk 1]
```diff
@@ -257,6 +257,10 @@
     static final ConfigKey<Long> VmJobCheckInterval = new ConfigKey<Long>("Advanced", Long.class, "vm.job.check.interval", "3000",
             "Interval in milliseconds to check if the job is complete", false);
 
+    static final ConfigKey<Boolean> VolumeUrlCheck = new ConfigKey<Boolean>("Advanced", Boolean.class, "volume.url.check", "true",
+            "Check the url for a volume before downloading it from the management server. Set to flase when you managment has no internet access.",
```
Comment (wido) [inline server/src/com/cloud/storage/VolumeApiServiceImpl.java]: Typo, flase instead of false

**Comment by rhtyd** [comment]
@DaanHoogland is this same as https://github.com/apache/cloudstack/pull/2232/files (or parts of it?)

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
@wido @rohit still lgty?

**Comment by rohit** [comment]
Folks I'm not the Rohit you'll are looking for.

**Comment by DaanHoogland** [comment]
sorry @rohit, nice to see you anyway

**Comment by DaanHoogland** [comment]
@rhtyd @wido restarted the blueoranutang, will merge if it looks succesful

**Pull request merged**